include/party_menu.h

```diff
@@ -33,7 +33,7 @@ struct Struct203CEC8
     u8 unk8_0:4;
     u8 mode:2;
     u8 unk8_2:2;
-    s8 unk9;
+    s8 unk9; //selected party index
```

Comment:
Should probably name the struct, or at least the field, rather than leaving a comment.

Reply:
I'm still not 100% certain on the purpose of the field. I should have put a question mark at the end of the comment honestly. The struct itself has stuff to do with the instance of the party screen shown, but I'm not certain how to put that into a name.